src/main/java/life/qbic/io/commandline/OpenBISPasswordParser.java

```diff
@@ -0,0 +1,40 @@
+package life.qbic.io.commandline;
+
+import java.io.Console;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class OpenBISPasswordParser {
+
+  private static final Logger LOG = LogManager.getLogger(OpenBISPasswordParser.class);
+
+  /**
+   * Retrieve the password from input stream
+   *
+   * @return The password
+   */
+  public static String readPasswordFromInputStream() {
+    char[] password;
+    Console console = System.console();
+    if (console == null) {
+      LOG.error(
+          "Could not get console instance!"
+              + " Please make sure that you're running this from a normal console, a console supplied by an IDE will not suffice!");
```

Comment:
Why not allow for any console or input? The method signature indicates that any InputStream would be fine.

Reply:
Not sure. This is legacy code that is now shown in this PR, because I applied google code style. Should have done that in a separate PR...